archive_converter_test.py

```diff
@@ -10,6 +10,13 @@ def test_generate_email_from_single_email_thread(self):
         self.assertEqual(email.from_, "Raul E Rangel <rrangel@chromium.org>")
         self.assertTrue(len(email.content) > 0)
         
+    def test_update(self):
+        archive_index = ArchiveMessageIndex()
+        archive_index.update('test_data')
+        size = archive_index.size()
+        self.assertEqual(size, archive_index.size())
```

Comment:
```suggestion
        old_size = archive_index.size()
        archive_index.update('test_data')
        self.assertEqual(archive_index.size(), old_size)
```

Perhaps also rename this test case to be more clear what it's testing